### PR TITLE
qa/tests - added skeleton for mimic point to point upgrades testing

### PR DESCRIPTION
**Pull request by yuriw**
Fixes https://tracker.ceph.com/issues/24316
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2018-06-25_21:11:03-upgrade:mimic-p2p-mimic-distro-basic-smithi/

File: qa/suites/upgrade/mimic-p2p/point-to-point-upgrade.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    conf:
+      mon:
+        mon debug unsafe allow tier with nonempty snaps: true
+        mon warn on pool no app: false
+      osd:
+        osd map max advance: 1000
+        osd_class_load_list: "cephfs hello journal lock log numops rbd refcount 
+                              replica_log rgw sdk statelog timeindex user version"
+        osd_class_default_list: "cephfs hello journal lock log numops rbd refcount 
+                                 replica_log rgw sdk statelog timeindex user version"
```
Comment (cbodley) [inline qa/suites/upgrade/mimic-p2p/point-to-point-upgrade.yaml]: add `otp` to both `osd_class_load_list` and `osd_class_default_list` to fix the errors related to `radosgw-admin mfa create`

**Comment by yuriw** [comment]
rerun with `otp` added pre @cbodley comment

http://pulpito.ceph.com/yuriw-2018-06-26_15:04:00-upgrade:mimic-p2p-mimic-distro-basic-smithi/

**Comment by yuriw** [comment]
@batrick do you how to fix this job:

 http://pulpito.ceph.com/yuriw-2018-06-26_19:27:38-upgrade:mimic-p2p-mimic-distro-basic-smithi/2707268

`2018-06-26T19:46:14.572 INFO:tasks.workunit.client.0.smithi192.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/suites/blogbench.sh: line 8: cd: too many arguments`

**Comment by yuriw** [comment]
more whitelsiting

http://pulpito.ceph.com/yuriw-2018-06-26_21:17:46-upgrade:mimic-p2p-mimic-distro-basic-smithi

**Comment by yuriw** [comment]
Note: one job failed due to this fix https://github.com/ceph/ceph/pull/22725

**Comment by neha-ojha** [comment]
nit: typo in commit message

**Comment by yuriw** [comment]
Fixed typo, thx

**Pull request merged**